pkg/registry/rbac/helpers.go

```diff
@@ -26,6 +28,10 @@ import (
 // IsOnlyMutatingGCFields checks finalizers and ownerrefs which GC manipulates
 // and indicates that only those fields are changing
 func IsOnlyMutatingGCFields(obj, old runtime.Object, equalities conversion.Equalities) bool {
+	if old == nil || reflect.ValueOf(old).IsNil() {
```

Comment:
We need the reflect based nil check?

Reply:
yes. some of the tests pass nil -> runtime.Object -> NewAdmissionAttribute() which ends up as a nil-referencing interface